### PR TITLE
[fix]: transform all hardcode addresses to lowercase

### PR DESCRIPTION
**Pull request by RDMStreet**

### CODE REVIEW

**Review by SergeyPoslavskiy — APPROVED** [review]

File: src/core/entities/Trade.spec.ts [hunk 1]
```diff
@@ -10,13 +10,13 @@ import { WeiAsToken } from './Wei'
 describe('Trade', () => {
   test('exact in', () => {
     const TokenA = new TokenImpl({
-      address: parseAddress('0xb9920BD871e39C6EF46169c32e7AC4C698688881'),
+      address: parseAddress('0xb9920bd871e39c6ef46169c32e7ac4c698688881'),
```
Comment (0x009922) [inline src/core/entities/Trade.spec.ts:13]: Tests don't care about exact addresses, here could be just ANY valid address. For what do you change them here?

File: src/core/utils.spec.ts [hunk 2]
```diff
@@ -4,12 +4,12 @@ import { formatAddress, parseAddress, isEmptyAddress, computeTransactionFee } fr
 import { Wei } from './entities'
 
 describe('Formatting address', () => {
-  const ADDRESS = parseAddress('0xae3a8a1D877a446b22249D8676AFeB16F056B44e')
+  const ADDRESS = parseAddress('0xae3a8a1d877a446b22249d8676afeb16f056b44e')
```
Comment (0x009922) [inline src/core/utils.spec.ts:7]: Same - why do you change address in tests?

**Review by 0x009922 — CHANGES_REQUESTED** [review]
It seems that this PR contains extra changes.

File: src/core/entities/Trade.spec.ts (hunk 1 above)
Comment (RDMStreet) [reply src/core/entities/Trade.spec.ts]: Ok, I've left change in only 1 test that failed with uppercase

File: src/core/utils.spec.ts (hunk 2 above)
Comment (RDMStreet) [reply src/core/utils.spec.ts]: Ok, I've left change in only 1 test that failed with uppercase

**Pull request merged**